examples/FaxDelete.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+require_once __DIR__ . "/vendor/autoload.php";
+
+$config = Dropbox\Sign\Configuration::getDefaultConfiguration();
+
+// Configure HTTP basic authorization: api_key
+$config->setUsername("YOUR_API_KEY");
+
+$faxDelete = new Dropbox\Sign\Api\FaxApi($config);
+
+try {
+    $faxDelete->deleteFax("[FAX_NUMBER]");
```

Comment:
```suggestion
$faxApi = new Dropbox\Sign\Api\FaxApi($config);

try {
    $faxApi->deleteFax("[FAX_NUMBER]");
```

`FaxApi::deleteFax` to be renamed to `faxDelete`